apps/io_tester/io_tester.cc

```diff
@@ -774,7 +776,7 @@ class unlink_class_data : public class_data {
             const auto flags = open_flags::rw | open_flags::create;
 
             file_open_options options;
-            options.extent_allocation_size_hint = fsize;
+            options.extent_allocation_size_hint = align_up<uint64_t>(fsize, extent_size_hint_alignment);
```

Comment:
It's not extent allocation that should be aligned by io-tester, but rather the file size itself

Reply:
Sure - fixed for all job types in io_tester :+1: